packages/base-map/src/index.js

```diff
@@ -67,7 +78,42 @@ class BaseMap extends Component {
 
   onLeftClick = e => {
     const { onClick } = this.props;
-    if (typeof onClick === "function") onClick(e);
+    callIfValid(onClick)(e);
+  };
+
+  forwardOne = (eventName, e) => {
+    const layer = this.overlays.find(child => child.props.name === e.name);
+    if (layer) callIfValid(layer[eventName])(e);
+
+    // Raise onOverlayAdded for this control.
```

Comment:
Comment should be rewritten since more than just onOverlayAdded could be dispatched here.

Reply:
I clarified the comment.